pkg/state/state.go

```diff
@@ -2849,6 +2863,12 @@ func (st *HelmState) flagsForDiff(helm helmexec.Interface, release *ReleaseSpec,
 	}
 	flags = st.appendPostRenderArgsFlags(flags, release, postRendererArgs)
 
+	skipSchemaValidation := false
+	if opt != nil {
+		skipSchemaValidation = opt.SkipSchemaValidation
+	}
+	flags = st.appendSkipSchemaValidationFlags(flags, release, skipSchemaValidation)
```

Comment:
helm-diff plugin dont support this flag now. so we should wait that helm-diff supprt that then we can merge this. @zhaque44

Reply:
ok thank you sounds good